egs/aishell/s10/local/run_chain.sh

```diff
@@ -123,20 +125,23 @@ if [[ $stage -le 12 ]]; then
 fi
 
 if  [[ $stage -le 13 ]]; then
-  echo "$0: creating phone language-model"
-  $mkgraph_cmd $dir/log/make_phone_lm.log \
-    chain-est-phone-lm \
-     "ark:gunzip -c $tree_dir/ali.*.gz | ali-to-phones $tree_dir/final.mdl ark:- ark:- |" \
-     $dir/phone_lm.fst || exit 1
-fi
+  echo "$0: Making Phone LM and denominator and normalization FST"
+  mkdir -p $dir/den_fsts/log
+
+  # We may later reorganize this.
+  cp $tree_dir/tree $dir/default.tree
 
-if [[ $stage -le 14 ]]; then
-  echo "creating denominator FST"
-  copy-transition-model $tree_dir/final.mdl $dir/0.trans_mdl
-  cp $tree_dir/tree $dir 
-  $train_cmd $dir/log/make_den_fst.log \
-    chain-make-den-fst $dir/tree $dir/0.trans_mdl $dir/phone_lm.fst \
-       $dir/den.fst $dir/normalization.fst || exit 1
+  echo "$0: creating phone language-model"
+  $train_cmd $dir/den_fsts/log/make_phone_lm_default.log \
```

Comment:
I guess it's better if you can define `lang="default"` at the top of this file and replace `default` with `${lang}`

Reply:
Done